### DISCUSSION TITLE
安装Alist的路由器重启后挂载的网盘全部掉盘

### DISCUSSION THREAD

**Discussion by Leefurmore** [AlistGo/alist#4533] [Q&A] [answered]
### Please make sure of the following things

- [X] I have read the [documentation](https://alist.nn.ci).
- [X] I'm sure there are no duplicate issues or discussions.
- [X] I'm sure it's due to `alist` and not something else(such as `Dependencies` or `Operational`).
- [X] I'm sure I'm using the latest version

### Alist Version / Alist 版本

git-24.147.45139-8c22904

### Driver used / 使用的存储驱动

Aliyun/Baiduyun

### Describe the bug / 问题描述

只要路由器重启，挂载的网盘必定掉，需要进入管理页面，存储选项卡上，点击重新加载才能正常使用，怎样才能不掉盘或者路由器重启后自动重新加载？

### Reproduction / 复现链接

内网链接

### Logs / 日志

<img width="1403" alt="iShot_2023-06-05_15 30 08" src="https://github.com/alist-org/alist/assets/112952363/3452a759-b231-4288-9ed5-6f3a79e32bbf">

**Comment by welcome** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by xhofe** [comment]
延时启动。

**Reply by Leefurmore** [reply]
> 延迟启动。

好的，在哪里能设置呢？我是在op上运行的。

**Reply by xhofe** [reply, marked as answer]
目前这个需要你自行探索，这个是与系统相关的，下个版本会在alist自身加一个这种设置。

**Reply by Leefurmore** [reply]
好的，我研究一下启动项参数，实在不行等下个版本，感谢及时回复！

**Reply by Hua-Fen** [reply]
找到延迟启动的方法了吗，我现在也是同样的问题

**Reply by anwen-anyi** [reply]
> 找到延迟启动的方法了吗，我现在也是同样的问题

配置文件有延时启动选项，自己设置 具体说明文档有写

**Reply by woontay** [reply]
延迟启动的文档说明，需要的可以看下  https://alist.nn.ci/zh/config/configuration.html#delayed-start
